Applications/GesturePod/onComputer/README.md

```diff
@@ -0,0 +1,46 @@
+GesturePod *(Gesture Recognition Pipeline)*
+=========================
+
+## Table of Contents
+
+- [About](#about)
+- [Quick Start](#quick-start)
+- [Docs](#docs)
+- [Also](#also)
+
+## About
+
+GesturePod is a plug-and-play, gesture recognition device. 
+
+As the GesturePod involves hardware, this implements GesturePod's gesture recognition pipeline on your system. We have stored sensor data collected from the GesturePod. This pre-stored sensor data is used for gesture recognition. You can now play around with GesturePod on your system!
```

Comment:
@ShishirPatil on your "system"? you mean PC or arduino?

Reply:
PC. Clarified in the latest commit.